gui/mozregui/global_prefs.py

```diff
@@ -1,33 +1,34 @@
-from PyQt4.QtCore import QSettings, QVariant
 from PyQt4.QtGui import QDialog
 
 from mozregui.ui.global_prefs import Ui_GlobalPrefs
 from mozregui import patch_requests
 
 from mozregression.network import set_http_session
+from mozregression.config import DEFAULT_CONF_FNAME, get_defaults
+from configobj import ConfigObj
 
 
 def get_prefs():
     """
     Return the global prefs as a dict.
     """
-    settings = QSettings()
+    settings = get_defaults(DEFAULT_CONF_FNAME)
     options = {}
     options['persist'] = \
-        str(settings.value("globalPrefs/persist",
-                           QVariant("")).toString()) or None
+        str(settings.get("globalPrefs/persist",
+                         "").toString()) or None
```

Comment:
So, you should remove the "globalPrefs/" part of the option name, now that we use the config file.

Also toString() is for QString, but now we are only using python str. The code should look like:

options['persist'] = setting['persist'] or None
options['http_timeout'] = settings['http-timeout']